content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md

```diff
@@ -13,64 +13,75 @@ further_reading:
 Below is a step by step guide to configure Azure AD as a SAML IdP within Datadog:  
 **Note**: an Azure AD Premium Subscription is required to set this up
 
-1. Navigate to `manage.windowsazure.com`
+#### Configuration 
 
-2. Go to the Active Directory sub menu
+1. Navigate to `https://portal.azure.com/`.
 
-3. Select the active directory you wish to use for SSO
+2. Navigate to the **Enterprise applications** service.
```

Comment:
I think there should be a step before this to navigate to the Azure Active Directory tab on the left hand menu, no?